cli/pkg/commands/images/sync.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2024. Open Data Hub Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package images
+
+import (
+	"github.com/opendatahub-io/ai-edge/cli/pkg/commands/common"
+	"github.com/opendatahub-io/ai-edge/cli/pkg/commands/flags"
+	"github.com/spf13/cobra"
+)
+
+var syncCmd = common.NewCmd(
```

Comment:
@devguyio As discussed this sometime back, In sync command can we include flags "imageId" and "version" then assign values to it. Directly passing values as arguments looks unclear on what exactly those values are.

Reply:
Done